src/tigerbeetle/benchmark_driver.zig

```diff
@@ -66,6 +60,13 @@ pub fn main(allocator: std.mem.Allocator, args: *const cli.Command.Benchmark) !v
     const addresses = args.addresses orelse &.{tigerbeetle_process.?.address};
     try benchmark_load.main(allocator, addresses, args);
 
+    if (tigerbeetle_process) |*p| {
+        const rusage = p.deinit();
```

Comment:
:thinking: looks like we won't deinit `TigerBeetle` process if `        const me = try std.fs.selfExePathAlloc(allocator);` throws above. 

What's the motivation for moving this code out of a defer? (and, if we do move it out, then we can `s/catch {}/try` I guess?)

Reply:
Good point! The `try` over `catch {}` is the reason why. To keep this code working on Windows, I'd have to move the `if (data_file_created) {` logic into the same `defer`. I'd be fine with that.

Reply:
:thinking: Ah, I guess the original smell here is that we are doing too much in the defer. I think we should change the code to

```
var tigerbeetle_process: ?TigerBeetleProcess = null;
defer if (tigerbeetle_process) |*p| _ = p.deinit();

...
...

const rusage = p.deinit();
p = null;
```